mysql/README.md

```diff
@@ -0,0 +1,233 @@
+## MySQL Example Usage
+
+The following examples assume `harbourbridge` has been added to your PATH
+environment variable.
+
+To use HarbourBridge on a MySQL database called mydb using mysqldump, run:
+
+```sh
+mysqldump mydb | harbourbridge -driver=mysqldump
+```
+
+The tool can also be applied to an existing mysqldump file:
+
+```sh
+harbourbridge -driver=mysqldump < my_mysqldump_file
+```
+
+To specify a particular Spanner instance to use, run:
+
+```sh
+mysqldump mydb | harbourbridge -driver=mysqldump -instance my-spanner-instance
+```
+
+By default, HarbourBridge will generate a new Spanner database name to populate.
+You can override this and specify the database name to use by:
+
+```sh
+mysqldump mydb | harbourbridge -driver=mysqldump -dbname my-spanner-database-name
+```
+
+HarbourBridge generates a report file, a schema file, and a bad-data file (if
+there are bad-data rows). You can control where these files are written by
+specifying a file prefix. For example,
+
+```sh
+mysqldump mydb | harbourbridge -prefix mydb. -driver=mysqldump
+```
+
+will write files `mydb.report.txt`, `mydb.schema.txt`, and
+`mydb.dropped.txt`. The prefix can also be a directory. For example,
+
+```sh
+mysqldump mydb | harbourbridge -prefix ~/spanner-eval-mydb/ -driver=mysqldump
+```
+
+would write the files into the directory `~/spanner-eval-mydb/`. Note
+that HarbourBridge will not create directories as it writes these files.
+
+To use the tool directly on a MySQL database called mydb, run
+
+```sh
+harbourbridge -driver=mysql
+```
+
+It is assumed that MySQL database supports information schema tables and
```

Comment:
"It is assumed that MySQL database supports information schema tables" -- we can drop this since we're specifically discussion MySQL and I think all supported MySQL version have information schema.

Side note: we use to have infoschema.go as a generic file (and so the comment in that file about assuming information schema is supported made sense). But I'm wondering if we should drop the comment in those files now since the inforschema.go files are under mysql/ and postgres/ and both of those DBs support infoschema.

Reply:
Sure, I had kept this statement to be in sync with the comment in `infoschema.go` files. 
Both Postgres and Mysql DB supports Information schema, so it would be reasonable to drop "assuming information schema.." comment from both package's `infoschema.go` file and from README as well.